ister_gui.py

```diff
@@ -2068,10 +2068,12 @@ def __init__(self, cur_step, tot_steps):
         super(SelectDeviceStep, self).__init__()
         self._clicked = None
         self._actions = None
-        self.disks = get_list_of_disks()
+        self.disks = []
         self.progress = 'Step {} of {}'.format(cur_step, tot_steps)
 
     def handler(self, config):
+        # just look every time, slow-loading devices may show up late
+        self.disks = get_list_of_disks()
```

Comment:
Seems fine, I just wonder if maybe we should have a button to refresh the device list instead/too?

Reply:
That would be a good addition considering it would a user could later plug in a USB device and want to refresh the screen.